media/js/firefox/whatsnew/whatsnew-54.js

```diff
@@ -2,58 +2,106 @@
+            'es': 'https://app.adjust.com/2uo1qc?campaign=whats_new&adgroup=Google_Play_Store&creative=badge_es',
+            'fr': 'https://app.adjust.com/2uo1qc?campaign=whats_new&adgroup=Google_Play_Store&creative=badge_fr',
+            'gb': 'https://app.adjust.com/2uo1qc?campaign=whats_new&adgroup=Google_Play_Store&creative=badge_gb',
+            'id': 'https://app.adjust.com/2uo1qc?campaign=whats_new&adgroup=Google_Play_Store&creative=badge_id',
+            'in': 'https://app.adjust.com/2uo1qc?campaign=whats_new&adgroup=Google_Play_Store&creative=badge_in',
+            'it': 'https://app.adjust.com/2uo1qc?campaign=whats_new&adgroup=Google_Play_Store&creative=badge_it',
+            'jp': 'https://app.adjust.com/2uo1qc?campaign=whats_new&adgroup=Google_Play_Store&creative=badge_jp',
+            'mx': 'https://app.adjust.com/2uo1qc?campaign=whats_new&adgroup=Google_Play_Store&creative=badge_es-mx',
+            'pl': 'https://app.adjust.com/2uo1qc?campaign=whats_new&adgroup=Google_Play_Store&creative=badge_pl',
+            'ru': 'https://app.adjust.com/2uo1qc?campaign=whats_new&adgroup=Google_Play_Store&creative=badge_ru'
+        },
+        updateStoreLinks: function(country) {
+            var newAppStoreURL = WNP54.getStoreURL(country, 'ios');
+            var newPlayStoreURL = WNP54.getStoreURL(country, 'android');
 
-    var form = new Mozilla.SendToDevice();
+            if (newAppStoreURL) {
+                $('#appStoreLink').attr('href', newAppStoreURL);
+            }
 
-    form.geoCallback = function(countryCode) {
-        // only take action on non-us country codes
-        if (countryCode && countryCode !== 'us') {
-            // make sure the user's countryCode is in our list
-            if (appStoreURLs.hasOwnProperty(countryCode)) {
-                $('#appStoreLink').attr('href', appStoreURLs[countryCode]);
+            if (newPlayStoreURL) {
+                $('#playStoreLink').attr('href', newPlayStoreURL);
             }
+        },
+        getStoreURL: function(country, platform) {
+            var newURL;
+
+            // only update non-US
+            if (country && country !== 'us') {
+                switch(platform) {
+                case 'ios':
+                    if (WNP54.appStoreURLs.hasOwnProperty(country)) {
+                        newURL = WNP54.appStoreURLs[country];
+                    }
 
-            if (playStoreURLs.hasOwnProperty(countryCode)) {
-                $('#playStoreLink').attr('href', playStoreURLs[countryCode]);
+                    break;
+                case 'android':
+                    if (WNP54.playStoreURLs.hasOwnProperty(country)) {
+                        newURL = WNP54.playStoreURLs[country];
+                    }
+
+                    break;
+                }
             }
+
+            return newURL;
+        },
+        init: function() {
+            var xhr = new XMLHttpRequest();
```

Comment:
Totally up to you and something I haven't tried using yet, but seeing as this is for FF 54, could you use `fetch` here instead of `XMLHttpRequest`? https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch